src/components/NotificationDialog.vue

```diff
@@ -151,7 +151,8 @@ export default {
                 "Splunk": "Splunk",
                 "webhook": "Webhook",
                 "GoAlert": "GoAlert",
-                "ZohoCliq": "ZohoCliq"
+                "ZohoCliq": "ZohoCliq",
+                "whapi": "Whapi",
```

Comment:
```suggestion
                "whapi": "WhatsApp (Whapi)",
```

Otherwise people won't find this ^^